### ISSUE TITLE
Open feed link in a new tab

### ISSUE THREAD

**Issue by Patclash** [ancestorak/brief#296]
New little bug : 
when I open a item link or a link in an item, it open in a new tab in background instead in foreground that was in the previous versions

**Comment by jtrulson** [comment]
I see this too.

**Comment by VillageCatDad** [comment]
I see it as well.  Both on Windows 10 and Mac OS, using FF57 on both.

**Comment by tanriol** [comment]
Before Firefox 57 it seemingly used to follow the user's preferences. However, reading those is not possible now, so some other way of handling this problem has to be found.

**Comment by Patclash** [comment]
Just add a small comment :
When I right click on an item or link > open in a new tab it open in a new tab foreground ....

**Comment by tanriol** [comment]
For me it opens in the background. This behavior can be configured in Preferences, so Brief should (somehow) support both options...

**Comment by Patclash** [comment]
Yes , but if I do a mouse middle click it open in background ... 
So not the same behavior as other link in tab

**Comment by VillageCatDad** [comment]
Correct, right clicking on the article headline will follow your FF preferences.  If you left click on the article headline, then the new tab always opens in the background, and no switch occurs.

**Comment by tanriol** [comment]
Note to self: some users want this to be independent from the Firefox-global setting.

**Comment by gabri25ele** [comment]
yes, Behavior is incorrect.
**left** click and **middle** click on a link must NOT be the same!
If a click normally (left button) I should see the new tab directly (foreground), and only with the middle button the tab in background.

**Comment by jtrulson** [comment]
I agree with @gabri25ele.  The old behavior on a left click would open a new tab and switch to it immediately, just like firefox is configured to do globally (in my setup).  This is probably the most irritating thing using brief for me, and has been broken since the new version was released.

I'd love to see it fixed some day.

**Comment by M1Aston** [comment]
> I agree with @gabri25ele. The old behavior on a left click would open a new tab and switch to it immediately, just like firefox is configured to do globally (in my setup). This is probably the most irritating thing using brief for me, and has been broken since the new version was released.
> 
> I'd love to see it fixed some day.

So would I. ;-)
